### PR TITLE
Merge temp_dev into dev

### PR DESCRIPTION
**Pull request by marvinkruse**

### CODE REVIEW

File: test/modules/fundingManager/RebasingFundingManager.t.sol [hunk 1]
```diff
@@ -46,7 +46,7 @@ contract RebasingFundingManagerTest is ModuleTest {
     uint internal constant DEPOSIT_CAP = 100_000_000e18;
 
     // Other constants.
-    uint8 private constant DECIMALS = 18;
+    //uint8 private constant DECIMALS = 18;  //Question : Is there any reason to hardcode the decimals for this fundingmanager to 18?
```
Comment (FHieser) [inline test/modules/fundingManager/RebasingFundingManager.t.sol]: Still a question left
Sounds like a zitzak question xD

File: test/modules/fundingManager/RebasingFundingManager.t.sol [hunk 2]
```diff
@@ -104,7 +104,7 @@ contract RebasingFundingManagerTest is ModuleTest {
     // Tests: Initialization
 
     function testInit() public override(ModuleTest) {
-        assertEq(fundingManager.decimals(), DECIMALS);
+        //assertEq(fundingManager.decimals(), DECIMALS);
```
Comment (FHieser) [inline test/modules/fundingManager/RebasingFundingManager.t.sol]: also outcommented this code
probably in connection with the comment above

**Review by FHieser — APPROVED** [review]
Except for the 2 comments i made this looks pretty good

File: test/modules/fundingManager/RebasingFundingManager.t.sol (hunk 1 above)
Comment (marvinkruse) [reply test/modules/fundingManager/RebasingFundingManager.t.sol]: As the token of the `RebasingFundingManager` is initialized with the same decimals as the `orchestratorToken` aka `collateralToken`, it is not hardcoded to 18 in the contract. So I don't see why it should be hardcoded here.

I will change the test to get the correct decimal value from the underlying token instead.

File: test/modules/fundingManager/RebasingFundingManager.t.sol (hunk 2 above)
Comment (marvinkruse) [reply test/modules/fundingManager/RebasingFundingManager.t.sol]: Same as above, yeah.

**Pull request merged**